src/coreComponents/mesh/ElementType.hpp

```diff
@@ -29,6 +29,7 @@ namespace geosx
  */
 enum class ElementType : integer
 {
+  Vertex,        ///< Single-node vertex element
```

Comment:
Can I ask what motivated the addition of the Vertex in GEOSX ?

Reply:
Some of the VTK files I tested actually contained every single geometric entity (cell, face, edge, vertex) as a separate CELL of corresponding VTK type (e.g. `VTK_VERTEX`). Since I refactored the code a bit to treat all cell types the same way and then discard groups of elements we're not interested in (e.g. 0-1D ones), it was more convenient to just add another element type in GEOSX so that I could return it from the conversion function `convertVtkToGeosxElementType`, instead of adding special cases, `if` branches, etc. 

I agree that it looks somewhat superficial, since we have no current use for it. Looking forward though, I'm wondering if something like Material Point Method (MPM) implementation in GEOSX will be interested at some point in importing particles and associated data from an external dataset.